internal/beater/server.go

```diff
@@ -191,13 +192,19 @@ func newServer(args ServerParams, listener net.Listener) (server, error) {
 		return server{}, err
 	}
 
-	otlpBatchProcessor := args.BatchProcessor
+	var otlpBatchProcessor modelprocessor.Chained
+
+	esClient, err := args.NewElasticsearchClient(elasticsearch.DefaultConfig())
+	if err != nil {
+		return server{}, err
+	}
+	mapFetcher := r8.NewMapFetcher(esClient)
+	size := 128
```

Comment:
I'm not sure what's the format of this value, though I'd like to mention that an Android map could easily get to over 10MB in size, so I wanted to mention this here in case we should provide a higher number in here.

Reply:
I believe this is `size` as in number of entries, not number of bytes of the data cached.